### DISCUSSION TITLE
Moving discussions out of other repositories

### DISCUSSION THREAD

**Discussion by Rumyra** [mdn/mdn-community#481] [Announcements]
Hey everyone - just a quick heads up: As per the guidelines introduced over a year ago, _this_ repository is the place for discussions. The core team will be moving all other discussions out from other repositories and putting them here. This reduces the overhead of making sure we're responding as they are all in the same place.

**Comment by Rumyra** [comment]
We've moved all discussions that were on the mdn/content repository to the content category here. We'll be triaging over the next few weeks. Apologies if folks were inundated with notifications because of that 🙏

**Reply by NaysKutzu** [reply]
why
